tools/vscode/README.md

```diff
@@ -20,6 +20,19 @@ C/C++ code completion.
 
 For developers in China, you might need to set `http_proxy`, `https_proxy`, `all_proxy` in advance.
 
+If you see the following error message: 
+```
+ERROR: vscode@docker-desktop:/workspaces/siddharth-envoy$ tools/vscode/refresh_compdb.sh
```

Comment:
Could you remove the username from this line?
```suggestion
ERROR: vscode@docker-desktop:/workspaces/envoy$ tools/vscode/refresh_compdb.sh
```

Reply:
removed in the most recent commit :)